### PR TITLE
python mailer

### PR DESCRIPTION
**Pull request by kphlex**

### CODE REVIEW

File: secrets_dev.yml [hunk 1]
```diff
@@ -6,3 +6,7 @@ canvas_pwd: canvas-docker
 rocket_user_id: <your token user id here>
 rocket_auth_token: <your token here>
 vm_pwd: fuzz # for anyone using the Virtual machine to run their docker containers
+# - Gmail account for Python Mailer -
+
+gmail_email: devtestingalerts@gmail.com
+gmail_pwd: hYuMo258hq!/
```
Comment (lcundiff) [inline secrets_dev.yml:12]: Do you want this email and password pushed to secrets dev? Just making sure.

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,49 @@
+[[source]]
```
Comment (lcundiff) [inline Pipfile:1]: What does this file do? It looks like requirements.txt since it shows all the packages.

File: src/mailer/alerts.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+src_dir = os.path.abspath(os.path.join(current_dir, ".."))
+p4hApp_dir = os.path.abspath(os.path.join(src_dir, ".."))
+sys.path.append(p4hApp_dir)
```
Comment (lcundiff) [inline src/mailer/alerts.py:7]:
```suggestion
p4hApp_dir = os.path.abspath(os.path.join(current_dir, "../../"))
sys.path.append(p4hApp_dir)
```
Would something like this be possible?

**Review by lcundiff — CHANGES_REQUESTED** [review]

**Review by mrbrown444 — COMMENTED** [review]
Pls can I use this project as a webmail smtp ,to send mails in bulk ... ?thanks

**Review by lcundiff — APPROVED** [review]

**Pull request merged**